### PR TITLE
Made global npm modules be executed as local npm modules

### PR DESCRIPTION
**Pull request by ChiaMineJP**
Additional commits for https://github.com/Chia-Network/chia-blockchain/pull/9717
As I noted in the above PR, even if `npm ci` replaces `npm install`, global npm modules version is still variable.
To fix the version of global npm modules, I made a choice that let global npm modules be executed as local npm modules.

### CODE REVIEW

**Comment by emlowe** [comment]
I like this idea, although we control the build systems but this is better really.

Do we want to change the user install script - `install-gui.sh`? It has some similar issues where it installs things globally I think.

**Comment by ChiaMineJP** [comment]
I've found global npm install in `install-gui.sh` as below.
```bash
if [ "$UBUNTU_PRE_2004" = "True" ]; then
	echo "Installing on Ubuntu older than 20.04 LTS: Ugrading node.js to stable."
	UBUNTU_PRE_2004=true # Unfortunately Python returns True when shell expects true
	sudo npm install -g n
	sudo n stable
	export PATH="$PATH"
fi
```

Umm. If I remember correctly, `n` package is `nodejs`/`npm` version switcher like `nvm`.
In addition to fix the version of `n`, `sudo n stable`  also brings version variability.
I think improving `install-gui.sh` should be handled in another PR.
A big change will be inevitable if we go deep about `install-gui.sh`.

**Comment by emlowe** [comment]
Yes that makes sense...
You are right,  `n` is a node package manager like `nvm`

**Comment by ChiaMineJP** [comment]
Umm. 'Linux ARM64 installer on Python 3.8' not working.
I'll look into the root cause.

image

**Comment by ChiaMineJP** [comment]
@emlowe @seeden @cmmarslender 
ARM64 installer issue has been now resolved. The root cause was corrupted npm cache folder.
Now all global npm modules can be installed and executed locally.
Please review and merge if it's good. (Currently base branch is set to `npm_install`)

**Review by emlowe — APPROVED** [review]
I like this.

File: .github/workflows/build-linux-arm64-installer.yml [hunk 1]
```diff
@@ -110,6 +110,11 @@ jobs:
       run: |
         sh install.sh
 
+    - name: Setup Node 16.x
```
Comment (cmmarslender) [inline .github/workflows/build-linux-arm64-installer.yml:113]: This shouldn't be necessary - node should be preinstalled in the container we use (see line 17 of this file) `container: chianetwork/ubuntu-18.04-builder:latest`

That container comes from here https://github.com/Chia-Network/build-images/blob/main/ubuntu-18.04/Dockerfile#L34

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: .github/workflows/build-linux-arm64-installer.yml (hunk 1 above)
Comment (ChiaMineJP) [reply .github/workflows/build-linux-arm64-installer.yml:113]: Yep. This should be removed. Making a new commit.

**Comment by ChiaMineJP** [comment]
Note: Failure on `LGTM analysis: Javascript` is already expected. This is resolved once LGTM see commits on `main` branch.
See https://github.com/Chia-Network/chia-blockchain/pull/9750

**Review by cmmarslender — APPROVED** [review]

**Review by seeden — APPROVED** [review]
LGTM thank you

**Pull request merged**